### PR TITLE
Rule fix: Add fixer for no-on-ready

### PR DESCRIPTION
**Pull request by edg2s**
Part of #115

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	],
 	"rules": {
 		"eslint-plugin/consistent-output": "off",
+		"eslint-plugin/fixer-return": "off",
```
Comment (jdforrester) [inline .eslintrc.json:13]: Can we inline this instead?

**Review by jdforrester — APPROVED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (edg2s) [reply .eslintrc.json:13]: I don't think it's a useful rule. Often fixers will only run in a subset of cases, it isn't an error. We already assert that the fixer does something via output tests, and code coverage.

Comment (edg2s) [reply .eslintrc.json:13]: If the rule asserted there was at least _one_ return statement, that might be useful.

Comment (edg2s) [reply .eslintrc.json:13]: Filed as https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/issues/102

**Pull request merged**